src/libraries/Microsoft.Extensions.DependencyInjection/tests/DI.External.Tests/Microsoft.Extensions.DependencyInjection.ExternalContainers.Tests.csproj

```diff
@@ -11,13 +11,10 @@
     <SignAssembly Condition="'$(TargetFramework)' == 'net461'">false</SignAssembly>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Compile Include="..\DI.Specification.Tests\**\*.cs" />
-  </ItemGroup>
-
   <ItemGroup>
     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.DependencyInjection\src\Microsoft.Extensions.DependencyInjection.csproj" SkipUseReferenceAssembly="true" />
     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.DependencyInjection.Abstractions\src\Microsoft.Extensions.DependencyInjection.Abstractions.csproj" SkipUseReferenceAssembly="true" />
+    <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.DependencyInjection.Specification.Tests\src\Microsoft.Extensions.DependencyInjection.Specification.Tests.csproj"/>
```

Comment:
Do the tests in `Microsoft.Extensions.DependencyInjection.Specification.Tests` still get executed in our CI when we P2P them?

Reply:
They are all abstract base classes. The derived classes in other assemblies are what execute them.